osu.Game.Rulesets.Catch/UI/CatchPlayfieldAdjustmentContainer.cs

```diff
@@ -10,15 +10,21 @@ namespace osu.Game.Rulesets.Catch.UI
 {
     public class CatchPlayfieldAdjustmentContainer : PlayfieldAdjustmentContainer
     {
+        private const float playfield_size_adjust = 0.8f;
+
         protected override Container<Drawable> Content => content;
         private readonly Container content;
 
         public CatchPlayfieldAdjustmentContainer()
         {
-            Anchor = Anchor.TopCentre;
-            Origin = Anchor.TopCentre;
+            // because we are using centre anchor/origin, we will need to limit visibility in the future
+            // to ensure tall windows do not get a readability advantage.
```

Comment:
I'm a bit lost with this comment (probably because I've barely read catch code) - why does anchor impact readability?

Reply:
The difference is if you make the window very tall. You will be able to see fruit earlier than intended. If you compare a release build or master to this PR, you should notice the difference (this PR will show more view range, compared to master).

Reply:
Oh, tall windows as in literal game windows, gotcha. Not sure why I interpreted it the way I did at first as it makes no sense in retrospect...